apps/charterafrica/src/payload/collections/Pages.js

```diff
@@ -35,7 +36,15 @@ const Pages = {
     {
       name: "blocks",
       type: "blocks",
-      blocks: [Hero, Ecosystem, Mooc, Partners, Resources, Spotlight],
+      blocks: [
+        Ecosystem,
+        Explainer,
```

Comment:
This works but I think creating Explainers (note the s for plural) block that wraps Explainer blocks is a much better design e.g.
```js
  slug: 'explainers',
  fields: [
    {
      name: 'explainers', // required
      type: 'blocks',
      minRows: 1,
      blocks: [
        Explainer
      ]
    }
  ]
```

Unless there is issue with `blocks` having `blocks` fields.

Reply:
<img width="976" alt="image" src="https://user-images.githubusercontent.com/39160236/215438081-965bbeae-d173-429b-afa4-9d0714f6f25e.png">

Reply:
Not sure what does that mean @koechkevin